go/core/logger/logger.go

```diff
@@ -26,10 +26,12 @@ import (
 func init() {
 	// TODO: Remove this. The main program should be responsible for configuring logging.
```

Comment:
Has this TODO been resolved? How does the log level get configured by the user?

Reply:
yes, just has been resolved I think

Reply:
yes, It is resolved, the user can set the log level like:

```go
if err := genkit.Init(ctx, &genkit.Options{LogLevel: slog.LevelWarn}); err != nil {
        log.Fatal(err)
}
```

Reply:
Hmm, this is a bit tangential to the logging. In theory, flows/etc can be used without calling `genkit.Init` which is mostly responsible for starting the servers, which can be sidestepped by starting up your own server and exposing the flows directly. We still want to configure the logging level appropriately in those cases. In JS, we pulled it out of Genkit instance config and it's a top-level/global function.

Reply:
I refactored it, now we can set the log level like:

```go
logger.SetLevel(slog.LevelError)
```

and the default log level now is: LevelDebug